examples/README.md

```diff
@@ -0,0 +1,33 @@
+# ink! code examples
+
+This folder contains a set of example contracts for ink!.
+
+Have a look at the different examples to better understand how to use ink! to build your own Substrate smart contracts.
+
+Besides teaching you how to write smart contracts, this folder is also meant for testing purposes. If you want to contribute to Parity ink!, please run the `check-examples.sh` script in the `./scripts` folder before submitting your PR.
```

Comment:
```suggestion
Besides teaching you how to write smart contracts, this folder is also meant for testing purposes. If you want to contribute to Parity ink!, please run the `check-workspace.sh` and `check-examples.sh` script in the `./scripts` folder before submitting your PR.
```

Reply:
`check-workspace.sh` doesn't have anything to do with this ./examples folder, that's why I didn't mention it here. `check-examples.sh` on the other hand is using it.

It is already mentioned in the general README , that both files in the ./scripts should be run before committing a PR.

Reply:
That's true but I was wondering on the whole section as a whole because we shouldn't regard it as standard practice that people start hacking on our examples and file PRs but this paragraph gave me the idea that this might be intended dev behaviour. So we might even want to consider removing this paragraph. I do not mean that devs should not file PRs for obvious improvements to our examples though.

Reply:
Removed.